src/renderer/components/Slider.vue

```diff
@@ -9,7 +9,7 @@
       :max="max"
       :value="initialValue"
       class="slider"
-      @input="$emit('newValue', $event.target.value)"
+      @input="$emit('new-value', $event.target.value)"
```

Comment:
that's just a naming convention, emitted events should be written in kebap-case but camel case works just fine, could be that my linter didn't detect it